### DISCUSSION TITLE
Question about initialization Unit 4.2(Part 1-3)

### DISCUSSION THREAD

**Discussion by jihobak** [Lightning-AI/dl-fundamentals#50] [Lecture videos and quizzes] [answered]
Unit. 4.2 Multilayer Neural Networks (Part 1-3)

Let's say activation func is relu and If we initialize weights and bias to zero, then all of hidden nodes have zero value after one forward propagation.

here's a question, Can I say all parameters(weights and bias) don't change even after backprop? because backprob is the change of multiplication and there's zero layer in the middle.

![image](https://github.com/Lightning-AI/dl-fundamentals/assets/10293523/778ee1bd-4aa6-4ee9-808e-d53ea62dbcc3)

**Comment by rasbt** [comment, marked as answer]
Yeah, that's a fair point. Since all values are 0, even the bias units, there won't be an update since all gradient are 0
